templates/single-quiz.php

```diff
@@ -68,14 +72,16 @@ class="<?php sensei_the_question_class(); ?>"
 						/**
 						 * Action inside before the question content on single-quiz page
 						 *
+						 * @since  1.9.0
+						 *
+						 * @hook sensei_quiz_question_inside_before
+						 *
+						 * @param {int} $the_question_id The question ID.
```

Comment:
```suggestion
						 * @param {int} $question_id The question ID.
```

It's a nitpick but it keeps the hooks consistent.